corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -554,6 +557,13 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             // Reconcile state of "select all" checkbox
             self.ui.selectAllCheckbox.prop("checked", !_.difference(self._allCaseIds(), self.selectedCaseIds).length);
         },
+        verifySelectedCaseIdsLessThanMaxSelectValue: function () {
+            if (this.selectedCaseIds.length > this.maxSelectValue) {
+                let errorMessage = "You have selected more than the maximum selection limit of " + this.maxSelectValue +
+                ". Please uncheck values to continue."
```

Comment:
nit:
Kind of reads like you are supposed to uncheck everything that is selected. Should it say something like "Please uncheck _some_ values to continue"?

Reply:
resolved here: [7a9fd27](https://github.com/dimagi/commcare-hq/pull/32416/commits/7a9fd2761c00d5d3194b6f6d665fad835c866054) (more feedback on wording is welcome this one was a little tricky to find the right level of detail)